packages/engine-server/lib/helpers.js

```diff
@@ -36,8 +38,24 @@ const transportToNotification = (transport) => ({
   transport,
 })
 
+const changeFormatOnPhoneNumber = (phoneNumber) => {
+  let phoneNumberData
+  switch (true) {
```

Comment:
I think I prefer if this was written as 

  let phoneNumberData
  if (phoneNumber.startsWith('+')) {
    phoneNumberData = new PhoneNumber(phoneNumber)
  } else if (phoneNumber.startsWith('07')) {
    phoneNumberData = new PhoneNumber(phoneNumber, 'SE')
  }
  return phoneNumberData.getNumber('e164').replace('+', '')